src/calculateRentalCost.js

```diff
@@ -28,7 +28,26 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const basicDiscount = 50;
+  const liteDiscount = 20;
+  const pricePerDay = 40;
+  let totalPrice = 0;
+
+  totalPrice += pricePerDay * days;
+
+  if (days >= 3 && days < 7) {
+    totalPrice -= liteDiscount;
```

Comment:
You should not change totalPrice in that case, you may return expression
```suggestion
    return totalPrice - liteDiscount;
```